vividus-soft-assert/src/test/java/org/vividus/softassert/model/SoftAssertionErrorTests.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.softassert.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.vividus.softassert.issue.KnownIssueType;
+
+class SoftAssertionErrorTests
+{
+    private final SoftAssertionError softAssertionError = new SoftAssertionError(new AssertionError());
+
+    @ParameterizedTest
+    @CsvSource({
+        "null,true,false", "Known issue,true,false",
+        "Known issue,false,true"})
+    void testKnownIssue(String status, boolean potentiallyKnown, boolean expectedResult)
```

Comment:
Does ’status’  influence on the test result?

Reply:
thank you, updated